src/ray/common/ray_config_def.h

```diff
@@ -247,7 +247,7 @@ RAY_CONFIG(uint64_t, raylet_death_check_interval_milliseconds, 1000)
 
 /// These are used by the worker to set timeouts and to batch requests when
 /// getting objects.
-RAY_CONFIG(int64_t, get_timeout_milliseconds, 1000)
+RAY_CONFIG(int64_t, check_signal_interval_milliseconds, 1000)
```

Comment:
```suggestion
RAY_CONFIG(int64_t, get_check_signal_interval_milliseconds, 1000)
```

nit, it's check signal during get.

Can you add a comment as well.

Reply:
updated

Reply:
also added comments